style.css

```diff
@@ -549,18 +549,62 @@ textarea {
     font-family: "Poppins", sans-serif;
   }
 
+  /* navbar section */
+  .nav-item {
+    display: block;
+  }
+
+  .nav-item a {
+    color: #344563;
+    font-weight: 500;
+  }
+
+  .nav-menu {
+    padding-right: 70px;
+  }
+
+  .hamburger {
+    display: none;
+  }
+
+  /* header section */
   .hero-section {
     background-image: url(assets/images/desktop-bg-img.png);
     background-repeat: no-repeat;
     background-size: cover;
     background-color: #fff;
+    padding-top: 118px;
   }
 
   .hero-section > div {
     margin: auto;
     width: 70%;
   }
 
+  /* work section */
+  .container {
+    display: flex;
+    flex-direction: row;
+    margin: 30px 142px;
+  }
+
+  .row-reverse {
+    flex-direction: row-reverse;
+  }
+
+  .desktop-img {
+    width: 100%;
+  }
+
+  .snap-shoot-one {
+    width: 100%;
+  }
+
+  .Project-one-header {
+    width: 100%;
+  }
+
+  /* About section  */
   .about-content {
     display: flex;
     flex-direction: row;
```

Comment:
- [ ] As I mentioned in my previous review, kindly fix the design of the about myself section to match the design of the [Figma template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=34%3A848). 😁

Right now the whole section is only using half of the area, it should use the whole width, this is how it looks for me:
![image](https://user-images.githubusercontent.com/20567503/178076991-8ad5b788-e4aa-43a5-a595-b19df6645a13.png)

And this is how the Figma design looks:
![image](https://user-images.githubusercontent.com/20567503/178044225-f29a7ae3-f04c-410f-acae-009ccc1e7693.png)

Reply:
@mirouhml Thank you very much. It just skipped my remembrance.